### PR TITLE
ref: Fix some ruff rules for private access (SLF)

### PR DESCRIPTION
**Pull request by cbornet**
Fixing but not enforcing at the moment since a lot of attributes can't be changed in Component

Renamed:

- Vertex._added_message
- Vertex._custom_component Vertex._built
- Vertex._built_object
- Vertex._built_result
- Vertex._data
- Vertex._raw_params
- Vertex._has_cycle_edges
- Vertex._build_params()
- Vertex._parse_data()
- Vertex._built_object_repr()
- Vertex._finalize_build()
- Graph._has_session_id_vertices
- StoreService._get()
- ChatService._async_cache_locks
- build_vector_store._is_cached_vector_store_checked

### CODE REVIEW

File: src/backend/base/langflow/api/utils.py [hunk 1]
```diff
@@ -134,12 +134,12 @@ def format_elapsed_time(elapsed_time: float) -> str:
 async def build_graph_from_data(flow_id: str, payload: dict, **kwargs):
     """Build and cache the graph."""
     graph = Graph.from_payload(payload, flow_id, **kwargs)
-    for vertex_id in graph._has_session_id_vertices:
+    for vertex_id in graph.has_session_id_vertices:
```
Comment (ogabrielluiz) [inline src/backend/base/langflow/api/utils.py:137]: You probably need to update the method `define_vertices_lists` because of this

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]
This is a dangerous one. hahaha

File: src/backend/base/langflow/api/utils.py (hunk 1 above)
Comment (cbornet) [reply src/backend/base/langflow/api/utils.py:137]: done

**Comment by cbornet** [comment]
> This is a dangerous one. hahaha

Yes, I'm counting on you to know the places where there are traps like `define_vertices_lists ` 😄 . And tests should also catch potential issues.

**Comment by cbornet** [comment]
Can't rename `Vertex._data` to `Vertex.data` as the latter already exists (and `Vertex.data` is equal to `Vertex._data["data"]` which is a bit confusing...). So I renamed it to `Vertex.full_data`.

**Review by ogabrielluiz — APPROVED** [review]
LGTM

**Pull request merged**